### ISSUE TITLE
FR: Support for ARM64

### ISSUE THREAD

**Issue by lindermayr** [orangecoding/fredy#99]
Most of my Docker containers run on Raspberry Pi or in a free instance of Oracle Cloud. Unfortunately, they only support ARM64, so your software always exits with this error:

```
ubuntu@docker1:~/docker/traefik$ docker pull ghcr.io/orangecoding/fredy:master
master: Pulling from orangecoding/fredy
no matching manifest for linux/arm64/v8 in the manifest list entries
```
I've found some instructions for adding multiple platform support, but in lack of programming skills, I cant do it on my own:
https://docs.docker.com/build/building/multi-platform/

Do you think it'd be possible to tweak your software to play nice with different architectures like ARM64? That'd be a huge help!

**Comment by orangecoding** [comment]
Interesting. I'll dig into this

**Comment by orangecoding** [comment]
@lindermayr That should do the trick. Wdyt?
https://github.com/orangecoding/fredy/pull/101

**Comment by orangecoding** [comment]
yeah I know. there is a test failure. Seems like a provider once again changed their structure. It's a cats and mice game. Let me fix this. I will then merge it and let you test it.

**Comment by orangecoding** [comment]
@lindermayr I merged my pr, docker build is done and images have been pushed. Can you try again?
